docker-auth.yaml

```diff
@@ -0,0 +1,5 @@
+username: assignmentexec
+password: AssignmentExec@2020
+repository: trial2
```

Comment:
why "trial2"?

Reply:
I was trying to build and push images multiple times. So its just a dummy name in yaml.